B_adding_pages.md

```diff
@@ -222,7 +222,15 @@ There are a couple of things to notice here. We pattern match against the params
 
 Within the body of the `show` action, we also pass a third argument into the render function, a key value pair where `:messenger` is the key, and the messenger variable is passed as the value.
 
-It's good to remember that the keys to the params [Dict](http://elixir-lang.org/docs/stable/elixir/Dict.html) will always be strings.
+Note that if the code in the body of the action needs access to the entire map of parameters, the `show` function can be defined this way instead:
```

Comment:
Maybe this would cover it:

Note: If the body of the action needs access to the full map of parameters bound to the `params` variable, in addition to the bound `messenger` variable, we could define `show/2` like this: